### DISCUSSION TITLE
È possibile rimuovere Warning per l'uso di Tor Browser?

### DISCUSSION THREAD

**Discussion by andreasoc** [globaleaks/globaleaks-whistleblowing-software#3804] [Q&A] [answered]
Salve
Intanto complimenti per la piattaforma, fatta molto molto bene.
Non ho trovato la possibilità, nel backend,  di togliere il warning per l'utilizzo consigliato di TOR.
![image](https://github.com/globaleaks/GlobaLeaks/assets/2852354/9cf3578e-f4e9-4df9-8155-cdaca156d459)

Io sto facendo il deploy  in azienda e per vari motivi vorrei togliere questo avviso che allerti troppi gli utenti.. che  si allerterebbero troppo e al 99% non sanno installare tor.. ed è pericolo anche farglielo installare.
Si può ? o devo mettere mano al codice? 
Grazie mille !

**Comment by denisp76** [comment, marked as answer]
Ciao,
vai su Impostazioni > Avanzate e attiva l'opzione 'Disabilita il privacy panel'. Nel caso tu voglia inserire un banner personalizzato puoi lasciare disattiva questa opzione e attivare la successiva 'Abilita privacy panel personalizzato' inserendo il testo nella casella di input che comparirà.

**Reply by andreasoc** [reply]
grazie mille non l'avevo vista come opzione! 
ciao

**Comment by alessandroarmellini** [comment]
Buongiorno. 
Mi accodo a questa discussione con una domanda: creando un privacy panel personalizzato, è possibile inserire il link al sito TOR in automatico? Per evitare di copiarselo a mano (e, nel caso cambi, evitare di avere un link errato).

Grazie in anticipo.
